packages/create-instance/create-slot-vnodes.js

```diff
@@ -18,22 +20,41 @@ function createVNodesForSlot (
   const el =
     typeof slotValue === 'string' ? compileToFunctions(slotValue) : slotValue
 
-  const vnode = h(el)
+  let vnode = h(el)
+  if (typeof slotValue === 'string') {
+    const vue = new Vue()
+    const _vue = new _Vue()
+    for (const key in _vue._renderProxy) {
+      if (!(vue._renderProxy[key])) {
+        vue._renderProxy[key] = _vue._renderProxy[key]
+      }
+    }
+    try {
+      // $FlowIgnore
+      vnode = el.render.call(vue._renderProxy, h)
```

Comment:
Do components without a render function work here?

For example:

```js
mount(TestComponent,{
  slots: { default: {
    template: '<div />'
  }}
})
```

Can you add a test for this kind of slot